### PR TITLE
Improve and update docs

### PR DESCRIPTION
**Pull request by raphinesse**
Improve docs and bring them up to date with the latest changes.

Note that there are more supported options than documented. As they've not been documented before, I considered them private for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,30 +25,58 @@
...
-This module is used for fetching modules from npm and gitURLs. It fetches the modules via `npm install`. It can also `npm uninstall` modules from a project.
+This package can be used to install and uninstall Node.js packages using npm.
 
-## Usage:
+## Usage
 
-### Fetching:
-```
-var fetch = require('cordova-fetch');
+### fetch
```
Comment (janpio) [inline README.md]: `fetch` instead of just "fetch" maybe?

File: README.md [hunk 2]
```diff
@@ -25,30 +25,58 @@
...
+-|-
+`spec` | A spec for the package to be installed (anything supported by `npm install`)
+`dest` | Location where to install the package
+`opts` | Additional options (optional)
+
+##### Options
+
+Option | Default | Description
+-|-
+`save` | `false` | Adds the package as  dependency to `package.json` iff `true`
```
Comment (janpio) [inline README.md]: double space and "iff"

File: README.md (hunk 1 above)
Comment (janpio) [reply README.md]: (same below for `uninstall`)

File: README.md (hunk 2 above)
Comment (raphinesse) [reply README.md]: [Iff](https://en.wikipedia.org/wiki/If_and_only_if) is not a typo. But given the fact that this mistake has been pointed out to me a few times before, I start to wonder whether I should continue to use it. :thinking: 

Would *If and only if* have been less irritating?

File: README.md (hunk 1 above)
Comment (raphinesse) [reply README.md]: Can do

File: README.md (hunk 2 above)
Comment (janpio) [reply README.md]: It would consist of words that I know - but that's my problem ;) TIL...

**Comment by raphinesse** [comment]
@janpio Are you OK with the last (cosmetic) change with the badges too?

**Comment by janpio** [comment]
Yep, good to merge.

(Noticed the options tables were broken, fixed it)

**Pull request merged**